hadoop-hdds/client/src/test/java/org/apache/hadoop/hdds/scm/client/HddsClientUtilsTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.hadoop.hdds.scm.client;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.ConnectException;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+
+class HddsClientUtilsTest {
```

Comment:
Ozone tests should be name `Test*`, not `*Test`.  The latter is not run by Maven:

https://github.com/apache/ozone/blob/3c9df5cf3292b2feb0ffa1aa9c254b1c51c8cb9a/pom.xml#L2022-L2024

Test class for `HddsClientUtils` exists, unfortunately it's in the wrong directory:

`hadoop-ozone/client/src/test/java/org/apache/hadoop/ozone/client/TestHddsClientUtils.java`

Created HDDS-9845 for relocating this test class to the right place.

For now, please add the new test case to the existing `TestHddsClientUtils`,